studiopress/atomic-blocks
# Improve editor style prefixes for 5.4

Update the style selectors to support the new `block-` prefixes in WP 5.4.

Fixes #329
Fixes #328

### How to test

1. Check out the branch and run the build. 
2. Add the pricing table to a page and check that it displays as expected. 
3. Add the Hero Header section and check that it applies a white color to the headings.

### Suggested changelog entry

- Update the style selectors to support the new block prefixes in WP 5.4.

### Notes
